lib/tasks/sul.rake

```diff
@@ -176,8 +189,8 @@ namespace :sul do
                           else
                             ''
                           end
-            sunet_list = pub.contributions.where("status in ('new','approved') and created_at > ?",
-                                                 Time.zone.parse(date_since)).map do |c|
+            sunet_list = pub.contributions.where("status in ('new','approved') and created_at > ? and created_at < ?",
```

Comment:
Because this query is repeated a bunch of times, you could consider making this a defined scope on the model and reusing it. But I don't think it's a big deal.

Reply:
good point, but a bit ambivalent, since i am pretty sure this query is only used for rake task reports and not in the core part of the app itself

Reply:
:shipit: